src/test/java/co/cask/hydrator/salesforce/plugin/source/batch/SalesforceRecordReaderTest.java

```diff
@@ -46,11 +46,11 @@ public void testTypes() throws Exception {
       "\"0061i000003XNcDAAW\",\"false\",\"220000.0\",\"2019-02-22T07:03:21.000Z\",\"2018-11-15\"";
 
     Schema schema = Schema.recordOf("output",
-                                    Schema.Field.of("Id", Schema.of(Schema.Type.STRING)),
-                                    Schema.Field.of("IsDeleted", Schema.of(Schema.Type.BOOLEAN)),
-                                    Schema.Field.of("ExpectedRevenue", Schema.of(Schema.Type.DOUBLE)),
-                                    Schema.Field.of("LastModifiedDate", Schema.of(Schema.LogicalType.TIMESTAMP_MILLIS)),
-                                    Schema.Field.of("CloseDate", Schema.of(Schema.LogicalType.DATE))
+                      Schema.Field.of("Id", Schema.of(Schema.Type.STRING)),
```

Comment:
why the change in indentation here? I think your IDE may not be configured with the CDAP style xml